carma_wm_ctrl/src/MapConformer.cpp

```diff
@@ -81,6 +82,7 @@ std::vector<lanelet::traffic_rules::TrafficRulesUPtr> getAllGermanTrafficRules()
     catch (const lanelet::InvalidInputError& e)
     {
       // Ignore participants which there is no generic rules for
+      ROS_INFO ("Ignoring participants which there is no generic rule for...");
```

Comment:
Make this a ROS_INFO_STREAM and pass in the participant_types[i] so we log which participant is being ignored.